JS/index.js

```diff
@@ -1,8 +1,8 @@
 import MilitaryPlane from './planes/MilitaryPlane';
 import PassengerPlane from './planes/PassengerPlane';
```

Comment:
Have you tried to run the code after refactoring?
your import statements are not updated after code update. Please update it